test/website/index.html

```diff
@@ -67,6 +67,11 @@
         const apps = await raygunClient.Applications.getAll("");
         console.log("Applications", apps);
       }
+
+      async function getIntegrations() {
```

Comment:
The function name could be `fetchIntegrations` as the previous get all requests have start with `fetch` instead of `get`